docs/DeveloperGuide.md

```diff
@@ -25,6 +40,159 @@
 |v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
 |v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
 
+### Main Menu
+This section describes the implementation of core main menu features.
+
+**API:** `AllOnUs.java`
+
+The menu is accessed through a call to static method main() of the AllOnUs class. 
+The Class Diagram below shows the overall structure of the application from the menu's point of view.
+The AllOnUs class and other classes that are coupled to the menu are therefore included.
+
+![](images/MenuFeaturesClassDiagram.png)
```

Comment:
Nice class diagram! Clear, concise, and complements the write up of the class. Details of other classes that are not necessarily important to this section are also omitted for better readability.